docs/conf.py

```diff
@@ -31,57 +32,72 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['sphinx.ext.autodoc',
-              'sphinx.ext.ifconfig',
-              'sphinx.ext.viewcode']
+extensions = [
+    "nbsphinx",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.napoleon",
+    "sphinx-jsonschema",
+    "sphinx.ext.doctest",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.todo",
+    "sphinx.ext.coverage",
+    "sphinx.ext.mathjax",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.githubpages",
+    "sphinx.ext.todo",
+    "sphinxcontrib.towncrier",
+    "sphinx_issues",
+    "sphinx-favicon",
+]
```

Comment:
I am not sure we want to enable all these extensions just because they are enabled in QCoDeS

Reply:
that's fair I'll remove and we can add them back in if needed